retroarch.cfg

```diff
@@ -343,14 +343,6 @@
 # 0 dB is normal volume. No gain will be applied.
 # audio_mixer_volume = 0.0
 
-#### Microphone
```

Comment:
Any reason why microphone stuff is being removed here?

Reply:
Idk why that's happening. Let me try getting the most up-to-date one and replacing it. I didn't do that on purpose sorry

Reply:
Maybe some issue with the merge? I can try to put back all the things removed

Reply:
Sure, go ahead